webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/httpsfilesystem/HttpsSeekableByteChannel.scala

```diff
@@ -1,15 +1,14 @@
 package com.scalableminds.webknossos.datastore.storage.httpsfilesystem
 
-import net.liftweb.common.{Box, Empty, Failure, Full}
-import net.liftweb.util.Helpers.tryo
-
 import java.net.URI
 import java.nio.ByteBuffer
 import java.nio.channels.SeekableByteChannel
-import java.nio.charset.StandardCharsets
 import java.nio.file.OpenOption
 import java.util
-import scalaj.http.{Http, HttpResponse}
+
+// Does not work with Java 8, works with Java 11
```

Comment:
I’d say this comment can be removed, as you also adapted the readme (and the dockerfiles come with java 11 currently)